pkg/volume/flexvolume/probe_test.go

```diff
@@ -286,7 +291,7 @@ func TestProberError(t *testing.T) {
 // Installs a mock driver (an empty file) in the mock fs.
 func installDriver(driverName string, fs utilfs.Filesystem) {
 	driverPath := filepath.Join(pluginDir, driverName)
-	fs.MkdirAll(driverPath, 0666)
+	fs.MkdirAll(driverPath, 0777)
```

Comment:
Was this just wrong and it didn’t matter because the driver was it using a real Fs?

Reply:
correct, the afero hid this problem.